content/acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -60,6 +60,134 @@ weight: 10
+
+  * `-1` — not specified, common `--tls_ca` value will be used.
+  * `0` — do not request client certificate, ignore it if received;
+  * `1` — request client certificate, but don't require it;
+  * `2` — expect to receive at least one certificate to continue the handshake;
+  * `3` — don't require client certificate, but validate it if client actually sent it;
+  * `4` — (default) request and validate client certificate.
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--redis_tls_client_ca=<filename>`
+
+  Path to additional CA certificate for Redis' certificate validation (overrides `--tls_ca`).
+  Empty by default.
+
+
+* `--redis_tls_client_cert=<filename>`
+
+  Path to server TLS certificate presented to Redis (overrides `--tls_cert`).
+  Empty by default.
+
+
+* `--redis_tls_client_key=<filename>`
+
+  Path to private key of the TLS certificate presented to Redis (overrides `--tls_key`).
+  Empty by default.
+
+
+* `--redis_tls_client_sni=<SNI>`
+
+  Expected Server Name (SNI) of the Redis instance. Will be used `--redis_host_port` value if is empty.
+  Empty by default.
+
+
+* `--redis_tls_crl_client_cache_size=<count>`
+
+  How many CRLs to cache in memory in connections to Redis.
+  Use `0` to disable caching. Maximum is `1000000`. Default is `16`.
+  Cache uses [LRU](https://en.wikipedia.org/wiki/Cache_replacement_policies#Least_recently_used_(LRU)) policy.
+
+
+* `--redis_tls_crl_client_cache_time=<seconds>`
+
+  How long to keep CRLs cached, in seconds for connections to Redis.
+  Use `0` to disable caching. Maximum is `300` seconds. Default is `0`.
+
+
+* `--redis_tls_crl_client_check_only_leaf_certificate={true|false}`
+
+  This flag controls behavior of validator in cases when Redis' certificate chain contains at least one intermediate certificate.
+
+  * `true` — validate only leaf certificate
+  * `false` — (default) validate leaf certificate and all intermediate certificates
+
+  This option may be enabled in cases when intermediate CAs are trusted and there is no need to verify them all the time.
+  Also, even if this flag is `false` but there is no CRL's URL configured and there is no CRL's URL in intermediate CA certificates,
+  these intermediate CAs won't be validated since we don't know which CRLs could be used for validation.
+
+
+* `--redis_tls_crl_client_from_cert=<policy>`
+
+  How to treat CRL's URL described in a certificate from Redis server
+
+  * `use` — try URL(s) from certificate after the one from configuration (if set)
+  * `trust` — try first URL from certificate, if it does not contain checked certificate, stop further checks
+  * `prefer` — (default) try URL(s) from certificate before the one from configuration (if set)
+  * `ignore` — completely ignore CRL's URL(s) specified in certificate
+
+  "URL from configuration" above means the one configured with `--tls_crl_*_url` flags.
```

Comment:
`--tls_crl_*_url` — where this is coming from? should we put a link here? i haven't found any `tls_crl_...` flags for acra-addzone